src/P2/slotheap.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+typedef unsigned char byte;
+
+struct SLOTHEAP {
+    int cb;
+    byte* ab;
+    struct SLOT* pslotFree;
```

Comment:
Remove `struct` keyword and move definition of SLOT struct above the definition of SLOTHEAP